modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/SearchHeader/components/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {faker} from '@faker-js/faker';
+import {fireEvent, render} from '@testing-library/react';
+import i18n from '../../../../../../common/I18n';
+import SearchBar from './';
+
+describe('render Search bar on Overview page', () => {
+	test('update onChange', () => {
+		const functionMock = jest.fn();
+		const fakerSearchOnChange = faker.name.firstName();
+
+		const {queryByPlaceholderText} = render(
+			<SearchBar onSearchSubmit={functionMock} />
+		);
+
+		const searchInput = queryByPlaceholderText(
+			i18n.translate('find-a-project')
+		);
```

Comment:
`testing-library` has `queryByPlaceholderText()` already implemented, you don't need to extract from the render

```suggestion
		render(<SearchBar onSearchSubmit={functionMock} />);

		const searchInput = screen.queryByPlaceholderText(
			i18n.translate('find-a-project')
		);
```

Another thing I would like to mention, if the SearchBar is a component that MUST be on the page, we should change `queryByPlaceholderText` to `getByPlaceholderText`

`get` functions returns an error if the element (node) we are trying to catch is not found
`query` functions returns `null` if it is not found

Reply:
Great observation, Aildson! Thank you. About query or getByPlaceholder already be implemented, I tried the way you suggest but I got an error saying that is not defined. That's way I extracted from the render